htdocs/css/main.css

```diff
@@ -538,49 +756,43 @@ th {
   }
 
   .col-1 {
-    position: relative;
-    padding-left: 10px;
-    padding-right: 10px;
+    padding-left: 7.5px;
+    padding-right: 7.5px;
     width: 16.66667%;
     float: left;
   }
 
   .col-2 {
-    position: relative;
-    padding-left: 10px;
-    padding-right: 10px;
+    padding-left: 7.5px;
```

Comment:
Need to Check _grid.scss component mixin in foundation 5.
The gutter and padding is not correct. Also our _grid.scss set up in structure too.
@ahomiya @rabbitwish